src/services/tests/mod.rs

```diff
@@ -1 +1,2 @@
-mod invalidate_caches;
\ No newline at end of file
+mod invalidate_caches;
```

Comment:
we are quite inconssitent on our module visibility :/

Reply:
I think that since we are testing "java-like" but the rust way is a but more clumped up this could maybe worthy of a larger scale refactor/code re-org?

Reply:
The inconsistency in module visibility has nothing to do with tests. We include similar modules sometimes with `super` visibility and sometimes without visibility (in many places, not just tests). This is probably due to auto imports in the ide ... Would still expect that this is consistent.

Reply:
Regarding test structure refactoring, I see no reason why we should do this.

Reply:
I think this comment is outdated I will resolve it